frontend/src/pages/PipelineDetailsV2.test.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render } from '@testing-library/react';
+import * as React from 'react';
+import { CommonTestWrapper } from 'src/TestWrapper';
+import { testBestPractices } from '../TestUtils';
+import PipelineDetailsV2 from './PipelineDetailsV2';
+
+testBestPractices();
+describe('PipelineDetailsV2', () => {
+  beforeEach(() => {
+    // Required by reactflow render.
+    window.ResizeObserver =
+      window.ResizeObserver ||
+      jest.fn().mockImplementation(() => ({
+        disconnect: jest.fn(),
+        observe: jest.fn(),
+        unobserve: jest.fn(),
+      }));
+  });
+
+  it('Render detail page with reactflow', async () => {
+    const { container } = render(
+      <CommonTestWrapper>
+        <PipelineDetailsV2></PipelineDetailsV2>
+      </CommonTestWrapper>,
+    );
+    expect(container.querySelector('.react-flow__pane')).not.toBeNull();
```

Comment:
class names are always implementation details. If possible, we should add a data testid to the element you want to select

Reply:
Sounds good, added data-testid for testing purpose.